vmdb/lib/workers/mixins/event_catcher_openstack_mixin.rb

```diff
@@ -14,7 +14,7 @@ def event_monitor_handle
       options[:duration] = self.worker_settings[:duration]
       options[:capacity] = self.worker_settings[:capacity]
 
-      options[:client_ip] = MiqServer.my_server.ipaddress
+      options[:client_ip] = @server.ipaddress
```

Comment:
I'm not sure I follow this...unless I call the reader similarly in the `worker_initialization` method.  In which case, it's basically the same as what I've got.  I mean, I could definitely add the reader, but I still need to initialize the ivar early, and not let it be a lazy initialization by calling the reader from inside the thread.